src/main.cpp

```diff
@@ -51,6 +51,11 @@ const char VERSION[10] = "0.0.2";   // um7_driver version
 // us to publish everything we have.
 const uint8_t TRIGGER_PACKET = DREG_EULER_PHI_THETA;
 
+enum OutputAxes
```

Comment:
Let's follow the [roscpp style guide](http://wiki.ros.org/CppStyleGuide#Enumerations)